prov/verbs/configure.m4

```diff
@@ -44,6 +44,17 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 				[verbs_rdmacm_happy=0])
 	      ])
 
+	AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
+			   [
+			    #include <infiniband/verbs_exp.h>
+			   ],
+			   [
+			    return (IBV_EXP_DEVICE_ATTR_ODP | IBV_EXP_DEVICE_ODP);
```

Comment:
I *think* you want (IBV_EXP_DEVICE_ATTR_ODP && IBV_EXP_DEVICE_ODP) - i.e. both are available.